sdk/storage/azure_storage_blob/src/clients/blob_container_client.rs

```diff
@@ -74,7 +74,6 @@ impl BlobContainerClient {
     /// # Arguments
     ///
     /// * `blob_name` - The name of the blob.
-    /// * `options` - Optional configuration for the client.
```

Comment:
Good clean up, but should there be an options? Do you do this in other languages? I would presume not - just use the same options, but wanted to ask to be sure.

Reply:
Good callout here- we should be using the same options (and functionally that is true), this was just a bad copy and paste of a docstring and I forgot to remove the param. I made the same mistake in `blob_service_client.rs` and also corrected it there.